packages/web/components/cards/stake-dashboard.tsx

```diff
@@ -14,72 +15,76 @@ export const StakeDashboard: React.FC<{
-      title={t("stake.dashboard")}
-      titleIcon={icon}
-      titleIconAction="www.google.com"
-      width="45"
-    >
-      <div className="flex w-full flex-row justify-between py-10">
-        <StakeBalances
-          title={t("stake.stakeBalanceTitle")}
-          dollarAmount={`${price}`}
-          osmoAmount={balance.toString()}
-        />
-        <StakeBalances
-          title={t("stake.rewardsTitle")}
-          dollarAmount="$26.89"
-          osmoAmount="20"
-        />
+    const dollarBalance = balance ? priceStore.calculatePrice(balance) : 0;
+
+    const icon = (
+      <div className="flex items-center justify-center text-bullish-500">
+        <div className="mr-2 flex self-center">
+          <Icon id="open-book" height="14px" width="14px" />
+        </div>
+        <span className="caption text-sm">{t("stake.learn")}</span>
       </div>
-      <ValidatorSquadCard
-        setShowValidatorModal={setShowValidatorModal}
-        validators={validators}
-        usersValidatorsMap={usersValidatorsMap}
-      />
-      <div className="flex h-full w-full flex-grow flex-row space-x-2">
-        <RewardsCard
-          title={t("stake.collectRewards")}
-          titleIconUrl="www.google.com"
-        />
-        <RewardsCard
-          title={t("stake.investRewards")}
-          titleIconUrl="www.google.com"
+    );
+
+    return (
+      <GenericMainCard title={t("stake.dashboard")} titleIcon={icon} width="45">
+        <div className="flex w-full flex-row justify-between py-10">
+          <StakeBalances
+            title={t("stake.stakeBalanceTitle")}
+            dollarAmount={`${dollarBalance}`}
+            osmoAmount={balance.toString()}
+          />
+          <StakeBalances
+            title={t("stake.rewardsTitle")}
+            dollarAmount={dollarRewards?.toString()}
+            osmoAmount={stakeableRewards?.toString()}
+          />
+        </div>
+        <ValidatorSquadCard
+          setShowValidatorModal={setShowValidatorModal}
+          validators={validators}
+          usersValidatorsMap={usersValidatorsMap}
         />
-      </div>
-    </GenericMainCard>
-  );
-};
+        <div className="flex h-full w-full flex-grow flex-row space-x-2">
+          <RewardsCard
+            title={t("stake.collectRewards")}
+            titleIconUrl="www.google.com"
+          />
+          <RewardsCard
+            title={t("stake.investRewards")}
+            titleIconUrl="www.google.com"
+          />
+        </div>
+      </GenericMainCard>
+    );
+  }
+);
 
 const StakeBalances: React.FC<{
   title: string;
-  dollarAmount: string;
-  osmoAmount: string;
+  dollarAmount?: string;
+  osmoAmount?: string;
 }> = ({ title, dollarAmount, osmoAmount }) => {
   return (
     <div className="flex w-full flex-col justify-center pl-10">
       <span className="caption text-sm text-osmoverse-200 md:text-xs">
         {title}
       </span>
-      <h3>{dollarAmount}</h3>
+      <h3>{dollarAmount || "$0"}</h3>
```

Comment:
We should avoid hardcoding dollars, in the case we want to support other currencies in the future. In our price store, it should be fairly easy to drop in other currencies in the future.